11ty/definitions/barbarian.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Barbarian
+slug: barbarian
+flag:
+  text: 'Colonial/Racist slur'
+  level: 'Avoid'
+defined: true
+speech: noun
+alt_words:
+  - cruel
+  - vicious
+  - obscene
+  - feral
+---
+someone who is "savagely cruel"; primitive ; unsophisticated
```

Comment:
I imagine this is from a dictionary but we should avoid "savagely" here as I would want to offer that connection as problematic. (I imagine you recognised this given it's in quotes). 

Also remove space after primitive and before semicolon.